src/storage/src/hummock/iterator/skip_watermark.rs

```diff
@@ -42,10 +47,14 @@ pub struct SkipWatermarkIterator<I> {
 }
 
 impl<I: HummockIterator<Direction = Forward>> SkipWatermarkIterator<I> {
```

Comment:
nits: since `SkipWatermarkIterator` is only used by compactor, how about moving `skip_watermark.rs` into src/hummock/compactor?

Reply:
Of course, I use a separate pr for it